### ISSUE TITLE
make not work - need help please

### ISSUE THREAD

**Issue by ghost** [google/cpu-check#4]
macOS 

```bash
$ make
Scanning dependencies of target utils
[  3%] Building CXX object CMakeFiles/utils.dir/utils.cc.o
[  7%] Linking CXX static library libutils.a
[  7%] Built target utils
Scanning dependencies of target silkscreen
[ 11%] Building CXX object CMakeFiles/silkscreen.dir/silkscreen.cc.o
/Users/user/Downloads/cpu-check/silkscreen.cc:75:31: error: call to 'Json' is ambiguous
    last_error = absl::StrCat(Json("position", k), ", ", Json("is", v), ", ",
                              ^~~~
/Users/user/Downloads/cpu-check/utils.h:27:13: note: candidate function
std::string Json(const std::string& field, int v);
            ^
/Users/user/Downloads/cpu-check/utils.h:28:13: note: candidate function
std::string Json(const std::string& field, uint64_t v);
            ^
/Users/user/Downloads/cpu-check/utils.h:29:13: note: candidate function
std::string Json(const std::string& field, double v);
            ^
1 error generated.
make[2]: *** [CMakeFiles/silkscreen.dir/silkscreen.cc.o] Error 1
make[1]: *** [CMakeFiles/silkscreen.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by scal444** [comment]
Thanks for the report - I think this is because we don't have a size_t overload and MacOS size_t is ambiguous for some reason, same as https://stackoverflow.com/questions/42004974/function-overloading-integer-types-and-stdsize-t-on-64-bit-systems.

I'll work on fixing this upstream and then push out our changes. If you want to unblock yourself in the meantime, you can add a size_t overload of the "Json" method to utils.cc or change the type of "k" at the call site.

**Comment by scal444** [comment]
I don't have access to a Mac to validate the fix. Would you mind checking if this branch compiles?
https://github.com/google/cpu-check/tree/size-t-macos-fix

**Comment by ghost** [comment]
err, this isn't the program I was looking for.
The branch compiles partially, I got it to work but im struggling making a PR and it's very late

**Comment by ghost** [comment]
I send a PR #5 tht fixed the issues I was getting...

**Comment by scal444** [comment]
Awesome! I'll try to upstream these changes and get it merged. You'll need a CLA with google to get this exact PR merged (instructions should be in the PR comment chain)-if you don't want that, I can copy the changes into a new PR.

**Comment by ghost** [comment]
yeah I don't mind, I just dont understand what is all about

**Comment by scal444** [comment]
>err, this isn't the program I was looking for.

So were you trying to compile the wrong project, then? In that case we'd be lucky someone with a Mac tried out our code to let us know we had issues :) LMK when/if you sign that CLA, and I'll get your change merged.

**Comment by ghost** [comment]
Yes by mistake, I wanted to find the architecture of edgeTPU for reverse engineering, something complicated...
